doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -0,0 +1,720 @@
+`.Axes.errorbar`. The documentation now accurately lists which properties are
+passed to `.Line2D`, rather than claiming that all keyword arguments are passed
+on.
+
+``streamplot`` can disable streamline breaks
+--------------------------------------------
+
+It is now possible to specify that streamplots have continuous, unbroken
+streamlines. Previously streamlines would end to limit the number of lines
+within a single grid cell. See the difference between the plots below:
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+
+    w = 3
+    Y, X = np.mgrid[-w:w:100j, -w:w:100j]
+    U = -1 - X**2 + Y
+    V = 1 + X - Y**2
+    speed = np.sqrt(U**2 + V**2)
+
+    fig, (ax0, ax1) = plt.subplots(1, 2, sharex=True)
+
+    ax0.streamplot(X, Y, U, V, broken_streamlines=True)
+    ax0.set_title('broken_streamlines=True')
+
+    ax1.streamplot(X, Y, U, V, broken_streamlines=False)
+    ax1.set_title('broken_streamlines=False')
+
+New axis scale ``asinh`` (experimental)
+---------------------------------------
+
+The new ``asinh`` axis scale offers an alternative to ``symlog`` that smoothly
+transitions between the quasi-linear and asymptotically logarithmic regions of
+the scale. This is based on an arcsinh transformation that allows plotting both
+positive and negative values that span many orders of magnitude.
+
+.. plot::
+
+    import matplotlib.pyplot as plt
+    import numpy as np
+
+    fig, (ax0, ax1) = plt.subplots(1, 2, sharex=True)
+    x = np.linspace(-3, 6, 100)
+
+    ax0.plot(x, x)
+    ax0.set_yscale('symlog')
+    ax0.grid()
+    ax0.set_title('symlog')
+
+    ax1.plot(x, x)
+    ax1.set_yscale('asinh')
+    ax1.grid()
+    ax1.set_title(r'$sinh^{-1}$')
+
+    for p in (-2, 2):
+        for ax in (ax0, ax1):
+            c = plt.Circle((p, p), radius=0.5, fill=False,
+                           color='red', alpha=0.8, lw=3)
+            ax.add_patch(c)
+
+``stairs(..., fill=True)`` hides patch edge by setting linewidth
+----------------------------------------------------------------
+
+``stairs(..., fill=True)`` would previously hide Patch edges by setting
+``edgecolor="none"``. Consequently, calling ``set_color()`` on the Patch later
+would make the Patch appear larger.
+
+Now, by using ``linewidth=0``, this apparent size change is prevented. Likewise
+calling ``stairs(..., fill=True, linewidth=3)`` will behave more transparently.
+
+Fix the dash offset of the Patch class
+--------------------------------------
+
+Formerly, when setting the line style on a `.Patch` object using a dash tuple,
+the offset was ignored. Now the offset is applied to the Patch as expected and
+it can be used as it is used with `.Line2D` objects.
+
+Rectangle patch rotation point
+------------------------------
+
+The rotation point of the `~matplotlib.patches.Rectangle` can now be set to
+'xy', 'center' or a 2-tuple of numbers.
+
+Axes.inset_axes flexibility
+---------------------------
```

Comment:
I don’t know how much rearranging you generally do at this stage, but I wonder if this should go under “Figure and Axes creation / management”